### ISSUE TITLE
Use Navtabs for multiple versions of documents, first pass

### ISSUE THREAD

**Issue by afred** [samvera/samvera.github.io#187]
**Background:** after asking Tom Johnson, the author of our current Jekyll theme, [documentation-jekyll-theme](https://github.com/tomjoht/documentation-theme-jekyll), if he had any ideas regarding how to handle versioning of documentation, he suggested we look at [Navtabs](http://idratherbewriting.com/documentation-theme-jekyll/mydoc_navtabs.html).

**Done when:**
  1. We take a first pass at seeing how Navtabs would work better for multiple versions than our current solution (which is to add multiple copies in the nav menu, and have a drop down selector in the upper-right corner of the page).
  1. New tickets are created for implementing Navtabs, OR stick with what we have.

**Comment by afred** [comment]
TL;DR - NavTabs aren't a good choice for presenting multiple versions of documentation pages because:

1. Content is added to tabs with Liquid's `include` method, which is designed to include repeatable content snippets, rather than full pages. As such, front matter is ignored for pages included with `include`. And without front matter, we lose a lot of capabilities with older versions of pages, such as adding older versions to the A-Z index, adding tags, adding categories, title, keywords, etc.

1. The current table of content functionality only applies for the 'current' page. Thus if you `include` other pages in tabs, then you may very likely end up with a TOC that does not apply to the content in the tab that you're currently viewing, which is super confusing.

I think these drawbacks are enough to say that using NavTabs would not be better than our current system.

However, while playing around and learning a bit more about Jekyll, I think I found some ways to make our current versioning feature better. Look for additional tickets.
